.github/workflows/release.yml

```diff
@@ -78,25 +78,31 @@ jobs:
       matrix:
         os: [macos]
         ruby: [2.7]
-        arch: [x64]
+        arch: [x86_64, arm64]
     runs-on: macos-latest
     steps:
       - uses: ruby/setup-ruby@v1
         with:
           ruby-version: ${{ matrix.ruby }}
+
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
       - name: Generate Gem artifacts
         run: |
-          python3 -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder')"
-          chmod a+x builder
-          ./builder build -p ${{ env.PACKAGE_NAME }} downstream
+          ruby -v
+          gem install bundler
```

Comment:
Here also we can drop this when not using builder.  The setup-ruby task will install bundler I'm pretty sure.